helm/datadog_checks/helm/data/conf.yaml.example

```diff
@@ -0,0 +1,44 @@
+## All options defined here are available to all instances.
+#
+init_config:
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
+instances:
```

Comment:
can we add an optional parameter to indicate the k8s namespaces we need to target? as a user I may not want the agent to list secrets in all namespaces, if the field is not configured we monitor all namespaces

Reply:
That's not a need we currently have, but it sounds like a reasonable feature request for a next iteration.